zip-0226.rst

```diff
@@ -302,30 +302,90 @@ Burn Mechanism Consensus Rules
-+---------------+------------+-------------------------+-----------------------------------------------------------------------------------+
-
-In terms of the Action size, the ZSA action size differs from the Orchard action size by 32 bytes (due to the addition of the :math:`\mathsf{AssetBase}`). This implies that the size goes from 820 bytes in the Orchard action to 852 bytes in the ZSA Action.
 
 ZSA Asset Burn Description
 --------------------------
 
-A ZSA Asset Burn description is encoded in a transaction as an of a AssetBurn type:
+A ZSA Asset Burn description is encoded in a transaction as an instance of an ``AssetBurn`` type:
 
-+-------+-----------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
-| Bytes | Name      | Data Type                   | Description                                                                                                               |
-+=======+===========+=============================+===========================================================================================================================+
-| 32    | AssetBase | byte[32]                    | For the Orchard-based ZSA protocol, this is the encoding of the Asset Base :math:`\mathsf{AssetBase}^{\mathsf{Orchard}}`. |
-+-------+-----------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
-| 8     | valueBurn | :math:`\{1 .. 2^{64} - 1\}` | The amount of the AssetType being burnt.                                                                                  |
-+-------+-----------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
++-------+---------------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| Bytes | Name          | Data Type                   | Description                                                                                                               |
++=======+===============+=============================+===========================================================================================================================+
+| 32    | ``AssetBase`` | ``byte[32]``                | For the Orchard-based ZSA protocol, this is the encoding of the Asset Base :math:`\mathsf{AssetBase}^{\mathsf{Orchard}}`. |
++-------+---------------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| 8     | ``valueBurn`` | :math:`\{1 .. 2^{64} - 1\}` | The amount of the Asset Base being burnt.                                                                                 |
```

Comment:
`The amount being burnt.` will be enough

Reply:
Done.